prog/dftb+/lib_common/coherence.F90

```diff
@@ -47,6 +49,21 @@ module dftbp_coherence
 #:endfor
   end interface toleranceCoherence
 
+  !> Check exact coherence of data across processor(s) with error handling 
```

Comment:
You could use the macros from prog/dftb+/include/error.fypp which turn error handling into an optional argument return or an error call (and will make API error return management simpler if refactored in future). Example of use in prog/dftb+/lib_timedep/pprpa.F90